app/components/back-office-app-services/func-applications-background-jobs.tf

```diff
@@ -18,7 +18,7 @@ module "applications_background_job_functions" {
   servicebus_topic_nsip_documents_id                                                  = var.servicebus_topic_nsip_documents_id
   app_service_plan_id                                                                 = azurerm_service_plan.back_office_functions_plan.id
   service_bus_namespace_name                                                          = var.service_bus_namespace_name
-  back_office_storage_account_host                                                    = var.document_storage_account_endpoint
+  back_office_storage_account_host                                                    = var.document_storage_domain
   back_office_storage_account_domain                                                  = var.document_storage_api_host

```

Comment:
I wonder if we need to clarify some naming here? host goes to domain, and domain goes to api_host!

Reply:
Oh that's not right, updated it now.